supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -257,6 +257,22 @@ object TesTask {
       backend_parameters = Option(backendParameters)
     )
   }
+
+  def makeTask(tesTask: TesTask): Task = {
```

Comment:
I first tried to add a `fromTesTask` to a `Task` companion object, but somehow that ended up interfering with finding the implicit JSON serializer for `Task`. 🙄